src/MarginTrading.SettingsService.SqlRepositories/Repositories/AssetPairsRepository.cs

```diff
@@ -30,14 +33,18 @@ public class AssetPairsRepository : IAssetPairsRepository
                                                  "[MatchingEngineMode] [nvarchar] (64) NULL, " +
                                                  "[StpMultiplierMarkupBid] float NULL, " +
                                                  "[StpMultiplierMarkupAsk] float NULL, " +
+                                                 "[IsSuspended] BIT NOT NULL DEFAULT 0, " +
+                                                 "[IsFrozen] BIT NOT NULL DEFAULT 0, " +
+                                                 "[IsDiscontinued] BIT NOT NULL DEFAULT 0, " +
                                                  "CONSTRAINT AP_Id UNIQUE(Id)" +
                                                  ");";
         
-        private static Type DataType => typeof(IAssetPair);
-        private static readonly string GetColumns = "[" + string.Join("],[", DataType.GetProperties().Select(x => x.Name)) + "]";
-        private static readonly string GetFields = string.Join(",", DataType.GetProperties().Select(x => "@" + x.Name));
+        private static PropertyInfo[] TypeProps => typeof(IAssetPair).GetProperties()
+            .Where(x => x.Name != nameof(IAssetPair.IsSuspended)).ToArray();//get rid of Suspended flag, it is handled separately
```

Comment:
It is a little bit weird and I am not sure that is necessary, because in case of update you use current value.

Reply:
Let it be, no harm here..